mkosi/distributions/__init__.py

```diff
@@ -19,3 +19,7 @@ def kernel_image(name: str) -> Path:
     @classmethod
     def cache_path(cls) -> List[str]:
         raise NotImplementedError
+
+    @classmethod
+    def remove_packages(cls, state: "MkosiState", remove: List[str]) -> None:
+        pass
```

Comment:
Shouldn't this throw an exception if `remove` is not empty? That way RemovePackages= will be refused if not implemented for a distro.

Reply:
Changed that to a `NotImplementedError`.